src/main/java/mate/academy/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.dto.user;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import mate.academy.validation.email.Email;
+import mate.academy.validation.password.FieldMatch;
+
+@FieldMatch(
+        field = "password",
+        fieldMatch = "repeatedPassword",
+        message = "Password and repeated password must be equal"
+)
+public record UserRegistrationRequestDto(
+        @Email
+        String email,
+        String password,
+        String repeatedPassword,
```

Comment:
why there is no validation for these fields?

Reply:
fixed